File: src/main/java/training/katas/number_of_people_in_the_bus/code.cs
namespace trainig.katas.number_of_people_in_the_bus
{
    using System;
    using System.Collections.Generic;
    using System.Linq;

    public class Kata
    {
        public static int Number(List<int[]> peopleListInOut)
        {
            // Happy Coding

            return peopleListInOut.Select(item => item[0]).Sum() - peopleListInOut.Select(item => item[1]).Sum();
        }
    }
}

File: src/main/java/training/katas/number_of_people_in_the_bus/code.kt

package training.katas.number_of_people_in_the_bus

fun people(busStops: Array<Pair<Int, Int>>): Int {
    // code here
    return busStops.map { (a, _) -> a }.sum() - busStops.map { (_, b) -> b }.sum()
}
